net-nds/phpldapadmin/files/phpldapadmin-1.2.6.2-r1-default-templates.patch

```diff
@@ -0,0 +1,102 @@
+--- phpldapadmin/lib/PageRender.php_orig	2020-11-20 11:57:35.293875838 +0100
```

Comment:
Please add relevant comments, links etc at the top of each patch.  They should contain at least a short description of the problem they are trying to solve and a link to the issue or upstream patch.

Reply:
I added some comments at the top of the patches.